### ISSUE TITLE
Membership inference attack overall auc is  0.519427

### ISSUE THREAD

**Issue by M-Ghorbel** [Koukyosyumei/AIJack#125]
I'm training to use your membership inference attack to evalute my own model but i started first by running the code you provided to be familiar with every line.

I got wierd results (i did not change anything in your code)

train_accuracy:  0.996
test_accuracy:  0.5535 (very low accuracy for cifar10)

and 

overall auc is  0.519427 (random reslut which is not expected for a model runningh without dp-sgd)

can you please explain what is wrong?

**Comment by Koukyosyumei** [comment]
Thank you for your report! I'll investigate this issue.

**Comment by M-Ghorbel** [comment]
thank you, ,please let me know when you solve the problem

**Comment by Koukyosyumei** [comment]
@M-Ghorbel 

I have almost finished the investigation. The test accuracy on CIFAR-10 seems low, but the original paper also reports similar results (around 0.6), indicating that the model is heavily overfitted on their training sets. Note that both our implementation nor the original work didn't use sophisticated SOTA models.

As for the second question, the implementation contains some bugs, like the position of positive estimations within the returned values from sklearn API.  I re-implemented this attack and made it compatible with sklearn and PyTorch. Also, I updated the [tutorial](https://koukyosyumei.github.io/AIJack/notebooks/aijack_membership_inference.html), which uses MNIST instead of CIFAR-10 since MNIST requires less computational power. You can run this notebook in google colab.

**Comment by M-Ghorbel** [comment]
Thank you, i will try it.

**Comment by OctopusJIA** [comment]
I am also trying the membership inference and flowing the tutorial on 4.1 you proposed, but I didn't find the mnist dataset used in your code.

`df = pd.read_csv("/content/sample_data/mnist_train_small.csv", header=None)`

Cloud you please tell me where I can get this dataset?

**Comment by OctopusJIA** [comment]
I used the mnist dataset downloaded from kaggle in csv format to run successfully.
